examples/getMe.js

```diff
@@ -11,4 +11,5 @@ const { TelegramClient } = require('../gramjs')
     await client.connect()
 
     console.log(await client.getMe())
+    await client.disconnect().then(() => process.exit(0))
```

Comment:
This will make it only work in node environments and not in browsers.

Reply:
Fair, I only had Node in mind for that example